delivery/models/db_models.py

```diff
@@ -46,6 +46,9 @@ class StagingOrder(SQLAlchemyBase):
     # The target path into which the file/directory will be moved
     staging_target = Column(String)
 
+    # The size of the staging order in bytes
+    size = Column(Integer)
```

Comment:
Looks good, but as I haven't used SQLAlchemy + SQLite before I just wanna double check that an `Integer` will be able to contain the huge byte values we will store?